source/features/follow-file-renames.tsx

```diff
@@ -30,13 +30,14 @@ async function linkify(button: HTMLButtonElement, url: GitHubURL): Promise<void
 	for (const file of (files as File[])) {
 		if (file[fromKey] === url.filePath) {
```

Comment:
Now `url` is only used for this, so `linkify` should probably just accept `filePath`

Also line 59 is lost so it should probably be moved to line ~33